Sources/BraintreePayPal/BTPayPalVaultEditResult.swift

```diff
@@ -0,0 +1,8 @@
+import Foundation
+
+/// A result of the Edit FI flow used to display a customers updated payment details in your UI
+/// - Warning: This feature is currently in beta and may change or be removed in future releases.
+public class BTPayPalVaultEditResult {
```

Comment:
Same here, could maybe drop `Vault` and use this result for checkout and vault edit FI flows in the future.